tensorflow/examples/udacity/1_notmnist.ipynb

```diff
@@ -183,15 +183,17 @@
       "source": [
         "num_classes = 10\n",
         "\n",
-        "def extract(filename):\n",
-        "  tar = tarfile.open(filename)\n",
+        "def maybe_extract(filename):\n",
```

Comment:
Good catch.
There are multiple ways to solve this but all of them seem like overkill. Removed this logic.